File: performance/cal.c
#include <stdio.h>

int cal(int n)
{
  if( n <=0 ) return 0;

  return n+cal(n-1);
}

int main(int argc,char*argv[])
{
  int total=0;

  int n=10000;
  

  total=cal(n);


  printf("%d\n",total);
}

File: performance/cal.go
package main

import "fmt"

var total int = 0
var n int = 10000

func cal(n int) int{
  if n <=0 {
    return 0
  } else{
    return n+cal(n-1)
  }
}


func main(){
  total=cal(n)
  fmt.Println(total)
}
